scipy/stats/stats.py

```diff
@@ -3152,8 +3171,8 @@ def f_oneway(*args):
 
     If each group is made of constant values, and
     
-    - Exists 2 groups with different values, function returns (np.inf, 0)
-    - All values in all groups are the same, function returns (np.nan, np.nan)
+    - There exists 2 groups with different values, the function returns (``np.inf``, 0)
```

Comment:
sorry to nitpick this but change: 
exists 2 groups => exist at least 2 groups

Reply:
Yeeees... but in "mathematical" sense these two sentences are equal, and the former was shorter that's why I went for it. Will do :)